landscape/lib/sysstats.py

```diff
@@ -104,17 +108,27 @@ class ThermalZone(object):
     def __init__(self, base_path, name):
         self.name = name
         self.path = os.path.join(base_path, name)
-        temperature_path = os.path.join(self.path, "temperature")
+        temperature_path = os.path.join(self.path, "temp")
         if os.path.isfile(temperature_path):
-            for line in open(temperature_path):
-                if line.startswith("temperature:"):
-                    self.temperature = line[12:].strip()
-                    try:
-                        value, unit = self.temperature.split()
-                        self.temperature_value = int(value)
-                        self.temperature_unit = unit
-                    except ValueError:
-                        pass
+            with open(temperature_path) as f:
+                line = f.readline()
+                if re.match(r'^\d+\s*$', line):
+                    self.temperature_value = int(line.strip()) / 1000
+                    self.temperature_unit = 'C'
+                    self.temperature = '{:.1f}{}'.format(
```

Comment:
This is just a nitpick, but based on tests in `landscape/lib/tests/test_sysstats.py` I think there should be a whitespace between value and unit.

Reply:
Fixed.